frontend/src/js/components/TicketCountSelector.tsx

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+import _ = require('underscore');
+import React = require('react');
+import Bootstrap = require('react-bootstrap');
+
+import Seat from './Seat.tsx';
+import {IShow} from "../../../../backend/src/show";
+import {ISection} from "../../../../backend/src/venue";
+import {IVenue} from "../../../../backend/src/venue";
+
+export interface ITicketCountSelectorProps {
+  active: boolean;
+  conflictingSeatIds: number[];
+  chosenSeatIds: number[];
+  reservedSeatIds: number[];
+  show: IShow;
+  venue: IVenue;
+
+  onSeatClicked: Function;
+}
+
+export default class TicketCountSelector extends React.Component<ITicketCountSelectorProps,any>{
+  constructor(props: any) {
+    super();
+    this.state = {error: null};
+  }
+
+  onAdd(section, event) {
+    var sectionSeatIds = _.values(this.props.venue.sections[section.id].seats).map((s) => s.id) // _.keys returns strings, we need ints
```

Comment:
this feels the same as `_.pluck([...].seats, 'id')`. Also, can be combined with the chain below if it makes it cleaner